fw/cache.c

```diff
@@ -2527,11 +2527,11 @@ static TfwCfgSpec tfw_cache_specs[] = {
 	{
 		.name = "cache_size",
 		.deflt = "268435456",
-		.handler = tfw_cfg_set_int,
+		.handler = tfw_cfg_set_long,
 		.dest = &cache_cfg.db_size,
 		.spec_ext = &(TfwCfgSpecInt) {
-			.multiple_of = PAGE_SIZE,
-			.range = { PAGE_SIZE, (1 << 30) },
+			.multiple_of = 2 * 1024 * 1024,
+			.range = { 2 * 1024 * 1024, (1UL << 37) },
```

Comment:
In the comment of the PR it says that the range is `[16MB; 128Gb]`. It's appears that either the comment is inaccurate or the line of code above it wrong and it should be `{ 16 * 1024 * 1024, (1UL << 37) }`

Reply:
Yeah, thank you! Fixed.